ycsb-mongodb/mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -318,11 +470,29 @@ public void init() {
                     System.exit(1);
             }
 
-            // encryption - FLE
-            boolean use_encryption = Boolean.parseBoolean(props.getProperty("mongodb.fle", "false"));
+            // sharded
+            isSharded = Boolean.parseBoolean(props.getProperty("mongodb.sharded", "false"));
+
+            // encryption - FLE or Queryable Encryption
+            boolean use_fle = Boolean.parseBoolean(props.getProperty("mongodb.fle", "false"));
+            boolean use_qe = Boolean.parseBoolean(props.getProperty("mongodb.qe", "false"));
+            boolean use_encryption = use_fle || use_qe;
+            encryptionType = (use_fle ? Encryption.FLE : (use_qe ? Encryption.QUERYABLE : Encryption.UNENCRYPTED));
```

Comment:
Suggestion: The chained ternary operator is a bit ugly / obfuscated, is the following clearer?

```
            if (use_fle && use_qe) {
                System.err.println("ERROR: mongodb.fle and mongodb.qe cannot both be true");
                System.exit(1);
            }
            encryptionType = Encryption.UNENCRYPTED; // Should be the default anyway but ... 
            if(use_fle) {
                encryptionType = Encryption.FLE;
            } 
            if(use_qe) {
               encryptionType = Encryption.QUERYABLE;
            }
```

Reply:
yes, accepted your suggestion.